contrib/docker/docker-entrypoint.sh

```diff
@@ -4,14 +4,18 @@ set -ex
 if [ "$#" -ne 0 ]; then
     exec "$@"
 elif [ "$SUPERSET_ENV" = "development" ]; then
-    superset worker &
+    celery worker --app=superset.sql_lab:celery_app --pool=gevent -Ofair &
     # needed by superset runserver
     (cd superset/assets/ && yarn && yarn run sync-backend)
     (cd superset/assets/ && yarn run dev) &
-    flask run -p 8088 --with-threads --reload --debugger --host=0.0.0.0
+    FLASK_APP=superset:app flask run -p ${SUPERSET_PORT} --with-threads --reload --debugger --host=0.0.0.0
 elif [ "$SUPERSET_ENV" = "production" ]; then
-    superset worker &
-    superset runserver -a 0.0.0.0 -w $((2 * $(getconf _NPROCESSORS_ONLN) + 1))
+    celery worker --app=superset.sql_lab:celery_app --pool=gevent -Ofair &
+    gunicorn -w $((2 * $(getconf _NPROCESSORS_ONLN) + 1)) \
+        --timeout 60 -b  0.0.0.0:${SUPERSET_PORT} \
+        --limit-request-line 0 \
+        --limit-request-field_size 0 \
+        superset:app
```

Comment:
@mistercrunch can you validate those are the correct commands? I have no idea :)

Reply:
command looks good